packages/sdk-manager/src/interfaces/index.ts

```diff
@@ -0,0 +1,14 @@
+import { EVMAssetTransfer } from '@buildwithsygma/sygma-sdk-core';
+
+export type SdkManagerStatus =
```

Comment:
should this type go into one of the files in the types folder?

Reply:
maybe there's no need for two folders, and just have a `types` directory with both types and interfaces?